tensorflow/python/ops/nn_ops.py

```diff
@@ -508,7 +521,8 @@ def __init__(self,
                build_op,
                filter_shape=None,
                spatial_dims=None,
-               data_format=None):
+               data_format=None,
+               fused=False):
```

Comment:
Since WithSpaceToBatch is not really specific to cuDNN or convolution, it would be better not to do any of the fused logic here.   See the comment below about calling _build_op_atrous directly.